src/docs/config/overview.md

```diff
@@ -54,10 +54,17 @@ enableCache: true
 
 The global script runs once before your library/app loads, so you can do things like setting up a connection to an external service or configuring a library you are using.
 
-The code to be executed should be placed within a default function that is exported by the global script. Ensure all of the code in the global script is wrapped in the function that is exported.
+The code to be executed should be placed within a default function that is exported by the global script. Ensure all of the code in the global script is wrapped in the function that is exported:
+
+```javascript
+export default function() { // or export default async function()
+  initServerConnection();
+}
+```
 
 The global script config option takes a file path as a string.
```

Comment:
Could move this line to be the first sentence of the first paragraph, so that the note about "async" is just below the example.